src/index.html

```diff
@@ -4,9 +4,548 @@
+              <a href="#" class="navigation__link">samsung</a>
+            </li>
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">SMARTPHONES</a>
+            </li>
+            <li class="navigation__item">
+              <a
+                href="#"
+                class="navigation__link"
+                data-qa="nav-hover"
+              >
+                LAPTOPS & COMPUTERS
+              </a>
+            </li>
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">GADGETS</a>
+            </li>
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">TABLETS</a>
+            </li>
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">PHOTO</a>
+            </li>
+            <li class="navigation__item">
+              <a href="#" class="navigation__link">VIDEO</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="main">
+      <section class="main__catalog">
+        <ul class="main__list">
+          <li class="main__list-item" data-qa="card">
+            <div class="card">
+              <div class="card__inner">
+                <div class="card__img">
+                  <img
+                    src="./images/imac.jpeg"
+                    alt="imac"
+                    class="card__img card__img--imac"
+                  >
+                </div>
+
+                <div class="card__name">
+                  <h2 class="card__description">
+                     APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+                  </h2>
+                </div>
+
+                <div class="card__code">
+                  Product code: 195434
+                </div>
+
+                <div class="card__conteiner">
+                  <div class="rate">
+                    <div class="rate__stars rate__stars--4">
+                      <div class="rate__star"></div>
+
+                      <div class="rate__star"></div>
+
+                      <div class="rate__star"></div>
+
+                      <div class="rate__star"></div>
+
+                      <div class="rate__star"></div>
+                    </div>
+
+                    <div class="rate__text">
+                      Reviews: 5
+                    </div>
+                  </div>
+                </div>
+
+                <div class="card__price">
+                  <div class="card__priceText">
+                     Price:
+                  </div>
+
+                  <div class="card__value">
+                    $2,199
+                  </div>
+                </div>
+
+                <div class="card__buy">
+                  <a
+                    href="#"
+                    class="card__button"
+                    data-qa="card-hover"
+                  >
+                    Buy
+                  </a>
+                </div>
```

Comment:
Div wrapper is also redundant here, add all styles for the a element (card__button class)